### PR TITLE
Feature/sphinx docs

### PR DESCRIPTION
**Pull request by arunchawla-NOAA**
**Description**

This PR adds Sphinx documentation in the docs directory. The documentation reflects the current information in the wiki. This addresses issue #9 

**Type of change**

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

The Sphinx documentation has been tested using Sphinx on a local Mac as well as connecting to ReadTheDocs (RTD). Have confirmed that the RTD page gets updated every time the repository is updated. Currently RTD points to feature/Sphinx-docs where this was developed. Once this PR is approved and successfully merged into develop then a consequent action will be taken to point to develop in the RTD project. 

NOTE: You can see the Sphinx output for feature/Sphinx-docs in https://global-workflow.readthedocs.io/en/latest/

  
**Checklist**

- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ X] My changes generate no new warnings
- [ N/A] New and existing tests pass with my changes
- [ X] Any dependent changes have been merged and published

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+# html_theme_options = {}
+html_theme_options = {"body_max_width": "none"}
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+html_context = {}
+
+def setup(app):
```
Comment (aerorahul) [inline docs/source/conf.py:92]:
```suggestion

def setup(app):
```

**Review by aerorahul — APPROVED** [review]
This looks great.
We will fix the failing pytests (they started failing recently).

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by XianwuXue-NOAA — APPROVED** [review]
Looks good to me

**Comment by KateFriedman-NOAA** [comment]
This is really great!

**Pull request merged**